common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -119,7 +121,7 @@ public static void setEnabled(boolean enabled) {
      * Returns {@code true} if resource leak detection is enabled.
      */
     public static boolean isEnabled() {
-        return getLevel().ordinal() > Level.DISABLED.ordinal();
```

Comment:
@ninja- what @Scottmitch says is that you changed the semantics of "isEnabled()" as a user can change the level on the fly via https://github.com/ninja-/netty/blob/cant_touch_this/common/src/main/java/io/netty/util/ResourceLeakDetector.java#L134 . So ENABLED can not be static final as a user may change the level later on.